src/app/lib/config/services/scotland.js

```diff
@@ -147,7 +147,19 @@ export const service = {
       lastUpdated: 'Last updated: ',
     },
     footer: {
+      trustProjectLink: {
```

Comment:
Scotland won't have a trust project link, so doesn't need it here or in the links array.

Reply:
`trustProjectLink` is now an empty object.